continuous_integration/install.sh

```diff
@@ -76,3 +76,4 @@ if [ -z "$NUMPY_VERSION" ]; then
 fi
 
 python setup.py install
+pip install nose-exclude
```

Comment:
> I think that this is actually the standard way to write doctest :)

Not a doctest expert but the crux of the problem is that the doctests are failing in dill, i.e. it doesn't have anything to do with this script or relative vs absolute imports:
```
git clone https://github.com/uqfoundation/dill
cd dill
nosetests --with-doctest
```
yields plenty of errors.